### PR TITLE
At 70

### PR DESCRIPTION
**Pull request by grantwhiteman**
Added ESLint to CI, ran ESLint and fixed all errors

### CODE REVIEW

**Review by benclayton98 — DISMISSED** [review]
LGTM

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+module.exports = {
+  "env": {
+    "commonjs": true,
+    "es2021": true,
+    "node": true,
+    "cypress/globals": true,
+    "jasmine": true
+  },
+  "extends": "eslint:recommended",
```
Comment (dearshrewdwit) [inline .eslintrc.js:9]: Sure. Why not.

File: .github/workflows/node.js.yml [hunk 2]
```diff
@@ -38,6 +38,7 @@ jobs:
     - run: npm ci
     - run: npm install cypress-mochawesome-reporter
     - run: npx sequelize-cli db:migrate
+    - run: npx eslint . --fix
```
Comment (dearshrewdwit) [inline .github/workflows/node.js.yml:41]: Let's do this step before running db:migrate - fail fast.

**Review by dearshrewdwit — CHANGES_REQUESTED** [review]
Nice. Move the CI step earlier to avoid unnecessary schema migrations.

Can you share in #atlas-general what this PR is, and once merged, how it will affect everyone, and what they need to do to avoid CI failing them on rule violations

**Review by dearshrewdwit — APPROVED** [review]
Remember to raise visibility to the teams about this

**Review by benclayton98 — APPROVED** [review]
LGTM

**Pull request merged**